content/en/docs/setup/additional-setup/cni/index.md

```diff
@@ -91,6 +77,32 @@ This race condition is mitigated by a "detect and repair" method.
 Please take a look at [race condition & mitigation](#race-condition--mitigation) section to understand the implication of this mitigation.
 {{< /tip >}}
 
+#### Helm-based install
+
+When [installing Istio using Helm](/docs/setup/install/helm/), it's possible to install the CNI as a standalone chart, by using:
+
+    {{< text syntax=bash snip_id=install_ingressgateway >}}
+    $ kubectl create namespace istio-ingress
+    $ helm install istio-ingress istio/cni -n istio-ingress --wait
+    {{< /text >}}
+Once the CNI is successfully installed, configure the istiod sidecar-injector to use the CNI by setting `istio_cni.enabled=true` in the istiod helm chart.
+
+#### Operator-based install (deprecated)
```

Comment:
I don't install via istioctl is deprecated.

Reply:
install with istioctl is not deprecated, instead maybe revise the description, or the command to just be something like `$istioctl install --set components.cni.enabled=true -y`